internal/cmd/export.go

```diff
@@ -90,6 +92,17 @@ var namespacedResources = []schema.GroupVersionResource{{
 	Resource: "serviceaccounts",
 }}
 
+// These "removed" GVRs are for making our CLI backwards compatible with older Kube versions.
```

Comment:
:pick: If I understand correctly, I think it might be more accurate to say:
```suggestion
// These "removed" GVRs are for making our CLI backwards compatible with older PGO versions.
```
Kubernetes 1.25 support is what required PGO to use `v1` versions of CronJobs and PodDisruptionBudgets instead of the `v1beta1` versions, per
https://kubernetes.io/docs/reference/using-api/deprecation-guide/#cronjob-v125
https://kubernetes.io/docs/reference/using-api/deprecation-guide/#poddisruptionbudget-v125

Reply:
Yep, that's definitely what I meant, I just "misspoke". Good catch.